cmd/wstail/tail.go

```diff
@@ -46,18 +54,28 @@ func tailer(ws *websocket.Conn, filter filter) {
 		log.Errorf("failed to read log directory: %v", err)
 		return
 	}
+
 	rotated := newRotatedLogs(dir, names)
 	log.Infof("rotated logs: %#v", rotated)
-	tailingDepth := []string{"-n", "+1"}
-	if filter.isEmpty() {
-		// Limit the output of an empty filter to last tailMaxDepth lines
-		tailingDepth = []string{"-n", fmt.Sprintf("%v", tailMaxDepth)}
-	}
 	var files []string
 	if rotated.Main != "" {
 		files = append(files, rotated.Main)
 	}
-	files = append(files, "-F", filePath)
+	files = append(files, "--follow", filePath)
```

Comment:
@a-palchikov with `--follow` you don't need to use man or help for command to understand what it means.